playbooks/roles/docker/tasks/docker-logs.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: get docker logs
+  set_fact:
+    container_logs: ""
+
+- name: get docker logs of container {{container_id}}
+  raw: docker logs {{container_id}}
+  register: container_logs
+  when: container_id is defined
+  delegate_to: localhost
+  ignore_errors: True
+
+- name: get docker logs using docker ps
+  # raw: docker logs $(docker ps -a -q --filter ancestor={{zowe_docker_image}}:{{zowe_docker_tag}})
+  raw: docker logs $(docker ps -a -q 2>&1 | head -n 1)
```

Comment:
This will show log of the first docker container. Maybe we can filter from the image name which should give you better chance to find the correct container.

BTW, with test run, do you find `container_id` could be empty?

Reply:
Does `docker ps -a -q --filter ancestor=` give you failure? If so, you may also need to fix `playbooks/roles/docker/tasks/uninstall.yml` since it's using similar command.

Reply:
container id exists , we call that from  `install-docker.yml`

empty case - when we evoke it from `show-logs.yml`  as sometimes we quite before reaching install-docker log
,at that level we have zowe_docker_url only not even zowe_docker_image and zowe_image_tag
, so kept it without filters

Reply:
In pipeline, 

Called here:
https://github.com/zowe/zowe-install-packaging/blob/b01af5d55ed4030ebd6192fbb899e94d53b07b4f/tests/installation/src/__tests__/basic/install-docker.ts#L45

Evoked Here:
https://github.com/zowe/zowe-install-packaging/blob/b01af5d55ed4030ebd6192fbb899e94d53b07b4f/playbooks/show-logs.yml#L26

Reply:
modified as per query provided @jackjia-ibm